flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -1441,6 +1468,59 @@ private String buildShowCreateTableRow(
         return sb.toString();
     }
 
+    /** Show create view statement only for views. */
+    private String buildShowCreateViewRow(
```

Comment:
This method is largely a duplicate of `buildShowCreateTableRow`. It'd be nice to refactor this a bit to extract some of the common logic, i.e. to format the columns.

Reply:
Good idea. 
Done.